packages/ui/src/components/badge.tsx

```diff
@@ -17,7 +17,7 @@ enum BadgesHoverStates {
 }
 
 const badgeVariants = cva(
-  'inline-flex items-center border transition-colors focus:outline-none focus:ring-2 focus:ring-ring focus:ring-offset-2',
+  'inline-flex items-center border !leading-none transition-colors focus:outline-none focus:ring-2 focus:ring-ring focus:ring-offset-2',
```

Comment:
```suggestion
  'inline-flex items-center border leading-none transition-colors focus:outline-none focus:ring-2 focus:ring-ring focus:ring-offset-2',
```
we can use this style without !important

<img width="1563" alt="Arc 2025-01-22 11 36 50" src="https://github.com/user-attachments/assets/c8b80ec4-8a72-44ef-a6fa-9fbb6c60b79d" />